### ISSUE TITLE
Crash when pasting options while no object is selected

### ISSUE THREAD

**Issue by Augustifolia** [fireclawthefox/DirectGuiDesigner#21]
If `self.selectedElement` is `None` (no element is selected) while pasting options the application crashes. Adding a check in the method `pasteOptions` in DirectGuiDesigner fixes the issue. Something like this:
```
    def pasteOptions(self):
        if self.copyOptionsElementInfo is None: return

        if self.selectedElement is None:  # if no object is selected
            base.messenger.send("showInfo", ["Please select an object to paste to"])
            return

        self.__copyOptions(self.copyOptionsElementInfo, self.selectedElement)
```

Do you prefer a new pull request for every issue? Or can I add several bugfixes to the same pull request?

**Comment by fireclawthefox** [comment]
Oh, that's something I thought I'd already fixed. Maybe it was in one of the other editors.
Regarding the PR, as long as you split it into individual commits, you sure can put them into one PR.

**Comment by Augustifolia** [comment]
Just pushed a fix for this to #19

**Comment by fireclawthefox** [comment]
PR merged, closing this now.
